### ISSUE TITLE
🔤 Update copy for work preferences section of the profile

### ISSUE THREAD

**Issue by gobyrne** [GCTC-NTGC/gc-digital-talent#11296]
## 🔤 Copy

Updating the work preference copy to:
- get rid of half sentences in the radio button group or checkbox group
- standardize on first person "I will" instead of "You will"
- removing quotes " and slashes /
- preparing for the location preferences updates in #9699

(some of) These changes will also need to be reflected in the application - profile, admin view - profile, snapshot, profile download (word), and csv download.

## 🕵️ Details

Specific changes:
- Update copy for "work duration" radio button group - Also update the display version
![image](https://github.com/user-attachments/assets/9831d2ea-d786-43d8-9dc7-ba2fd5971184)

- Update copy for "job context" button group and title
![image](https://github.com/user-attachments/assets/c83edc00-8b3e-4e85-b13b-9969253132c4)

- **New subsection "Your current location"**
- Move "Province/Territory/Region" from the contact information into this work preference section and rename it to "Province, territory or region"
- Likewise for "City/Town" and renamed to "City or town"
![image](https://github.com/user-attachments/assets/66415e00-75a1-42ad-9198-97c8548ad73d)

- **New "Work location preferences" subsection**  (Everything in this section gets updated in #9699, we're just getting it ready in this ticket)
- Move "I am willing to work..." here, but leave copy the same
- Move "Please indicate if there is a city that you ..." here, but leave copy the same

![image](https://github.com/user-attachments/assets/9dbd95cc-109a-4615-91ec-5b0c33741872)

**Implications for similar sections across the platform:**
**Application**
Please make the same changes in the application profile section as to the work preferences section in the profile itself.

**Admin view of profile**
Please make the following updates to the 'Work preferences' section in the admin view of the user profiles:
-	Change ‘I would consider accepting a job that lasts for:’ to ‘**Contract duration preference**’ 
-	Change ‘I would consider accepting a job that:’ to ‘**Acceptable job requirements**’
-	Change the copy of the answers to job requirements to follow the format of ‘I am willing to work occasional overtime.’ Don’t use bold text in the answers.
-	Please put a new subsection under 'Work preferences' called '**Current location**' and include the city and region.
-	Please move the 'Work location' info under 'Work preferences'
-	Please bold the text ‘**Work location:**’ and ‘**Work location exceptions:**’ and make the answers normal text, no bold.

**Snapshot**
For the profile snapshot, in the work preferences section, please make the following updates:
-	Change ‘Job duration’ to ‘Contract duration preference’
-	Change the answers to ‘Any duration  (short term, long term, indeterminate)’ and ‘Indeterminate (permanent only)’
-	Change ‘Job contexts’ to ‘Acceptable job requirements’
-	Please change the answers to the format of ‘I am willing to work […]’
-	Please include a new subsection called ‘Current location’ after the acceptable job requirements and move the city and region info here.

**CSV**
To keep the order the same as on the platform, some columns need to be reordered. After priority number it should be:
-	Accept temporary
-	‘Would consider accepting a job that:’ which should be renamed to ‘Acceptable job requirements’
-	Location preferences
-	Location exemption

**Profile download (word)**
- Under the heading 'Work preferences', please include a subheading called ‘Contract duration preference’
- Change ‘Would consider accepting a job that:’ to ‘Acceptable job requirements’
- Change the format of the answers under ‘Acceptable job requirements’ to match what we have on the platform, for example ‘I am willing to work shift work.’ Instead of ‘Availability, willingness and ability to work shift-work.’
- Please put a new subsection under 'Work preferences' called 'Current location' and move the city and region info which currently under 'Contact information' here.
- Please move the 'Work location' info and make it a subsection under 'Work preferences'. Please change the copy to say 'Work location preferences'

![image](https://github.com/user-attachments/assets/f088cbd6-958e-4fcd-913f-3b54416d2b80)

**Find talent (talent search)**
No change for now.

## 🎨 Design File

https://www.figma.com/design/guHeIIh8dqFVCks310Wv0G/Component-library?node-id=2080-22407&t=BSfkpWur4ArAQj7X-0

### 🧑‍🎨 Designer
@NienkeBr  

## 🌎 Localization

(optional) Provide any new copy along with translations available.
NB: Translations:
- Contract duration preference -> Préférence pour la durée du contrat
- Acceptable job requirements -> Exigences de poste acceptables
-	J’accepte de faire des **heures supplémentaires à l’occasion**.
-	J’accepte de faire des **heures supplémentaires régulièrement**.
-	J’accepte de travailler **par quarts**.
-	J’accepte de travailler **par quarts et d’être sur appel 24 heures sur 24, 7 jours sur 7**.
-	J’accepte d’avoir un emploi qui **nécessite des déplacements**.
-	J’accepte d’avoir un emploi dans le cadre duquel je dois transporter, **soulever et déposer de l’équipement pesant jusqu’à 20 kg**.
-	J’accepte d’avoir un emploi dans le cadre duquel je dois **détenir un permis de conduire valide** ou profiter d’une mobilité personnelle qui est normalement associée à la possession d’un permis de conduire valide.
- Your current location -> Votre lieu de résidence actuel
- Work location preferences -> Préférences pour le lieu de travail

## ✅ Acceptance Criteria

- [ ] Work duration section updated to Contract duration preference
- [ ] Job context section updated to Acceptable job requirements
- [ ] New subsection "Your current location" and move the region and city fields here from the contact info
- [ ] New work location preferences and move the two remaining fields "Job locations" and "Location exclusions"
- [ ] Update CSV and profile downloads to reflect new text and organization

**Comment by gobyrne** [comment]
@NienkeBr can you look at the corresponding copy on:

- CSV & Profile downlaod
- Admin view of profile & snapshot
- Application
- Profile (is this one)
- Talent search (but facing managers, probably not)

**Comment by NienkeBr** [comment]
Updated the ticket to include necessary changes for the corresponding copy in the other sections. Let me know if you have questions.

**Comment by gobyrne** [comment]
I'm a little worried about the duplication of this copy across the platform. Ideally we would have only two sets of copy, one worded for applicants and one for managers.

Component-wise, I feel like the "snapshot", application and "profile" should be using the same.
